runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java

```diff
@@ -254,6 +254,10 @@
 	DRAGONSTONE(Type.GEM, "Dragon", "DR", ItemID.UNCUT_DRAGONSTONE, ItemID.DRAGONSTONE),
 	ONYX(Type.GEM, "Onyx", "ON", ItemID.UNCUT_ONYX, ItemID.ONYX),
 	ZENYTE(Type.GEM, "Zenyte", "Z", ItemID.UNCUT_ZENYTE, ItemID.ZENYTE),
+	SHADOW_DIAMOND(Type.GEM, "Shadow Diamond", "SHD", ItemID.SHADOW_DIAMOND),
+	BLOOD_DIAMOND(Type.GEM, "Blood Diamond", "BD", ItemID.BLOOD_DIAMOND),
+	ICE_DIAMOND(Type.GEM, "Ice Diamond", "SD", ItemID.ICE_DIAMOND),
```

Comment:
Why is the short name `"SD"` for the ice diamond?

Reply:
I had a typo. thanks for catching that. I pushed a fix